monai/transforms/lazy/functional.py

```diff
@@ -76,30 +79,28 @@ def apply_transforms(
     cumulative_xform = affine_from_pending(pending[0])
     cur_kwargs = kwargs_from_pending(pending[0])
     override_kwargs: dict[str, Any] = {}
-    if mode is not None:
-        override_kwargs[LazyAttr.INTERP_MODE] = mode
-    if padding_mode is not None:
-        override_kwargs[LazyAttr.PADDING_MODE] = padding_mode
-    if align_corners is not None:
-        override_kwargs[LazyAttr.ALIGN_CORNERS] = align_corners
-    if resample_mode is not None:
-        override_kwargs["resample_mode"] = resample_mode
-    override_kwargs[LazyAttr.DTYPE] = data.dtype if dtype is None else dtype
```

Comment:
there's a difference in providing `dtype=None` vs not providing this option (default float64)